blazar/cmd/manager.py

```diff
@@ -30,14 +31,23 @@
 from blazar.utils import service as service_utils
 
 
+class ManagerServiceSingleton:
```

Comment:
@diurnalist Any suggestion for the best pattern for this class? I have it returning the partial function if a resource type is passed in (for host, network, fip), but this pattern cannot be used for leases. And so it is returning 2 different data types, which is a bit confusing.